warbl2_firmware/Functions.ino

```diff
@@ -2985,11 +2987,14 @@ void saveCalibration() {
 
 // Load the stored sensor calibrations from EEPROM
 void loadCalibration() {
-    for (byte i = 0; i < 9; i++) {
-        byte high = readEEPROM((i + 9) * 2);
-        byte low = readEEPROM(((i + 9) * 2) + 1);
+    // for (byte i = 0; i < 9; i++) {
+    //     byte high = readEEPROM((i + 9) * 2);
+    //     byte low = readEEPROM(((i + 9) * 2) + 1);
+    for (byte i = EEPROM_SENSOR_CALIB_START; i < EEPROM_SENSOR_CALIB_START+18; i+2) {

```

Comment:
This isn't quite right i+2 should be i+=2, at least... but the whole thing may need a once-over to make sure it is equivalent too...

Reply:
You're right, thank you!